src/renderer/base/FontInfoBase.cpp

```diff
@@ -60,38 +56,38 @@ bool FontInfoBase::WasDefaultRasterSetFromEngine() const
     return _fDefaultRasterSetFromEngine;
 }
 
-LONG FontInfoBase::GetWeight() const
+unsigned int FontInfoBase::GetWeight() const
 {
-    return _lWeight;
+    return _weight;
 }
 
-PCWCHAR FontInfoBase::GetFaceName() const
+std::wstring FontInfoBase::GetFaceName() const

```

Comment:
Is the copy required or can most things instead make due with either a ref or a std::wstring_view?